.github/CHANGELOG.md

```diff
@@ -296,6 +296,20 @@
 
 <h3>Breaking changes</h3>
 
+* Updated how parameter-shift gradient recipes are defined for operations, allowing for
+  gradient recipes that are specified as an arbitrary number of terms.
+  [(#909)](https://github.com/PennyLaneAI/pennylane/pull/909)
+
+  Previously, `Operation.grad_recipe` was restricted to two-term parameter-shift formulas.
+  With this change, the gradient recipe now contains elements of the form
+  <img src="https://render.githubusercontent.com/render/math?math=[c_i, a_i, s_i]">, resulting in a gradient recipe of
+  <img src="https://render.githubusercontent.com/render/math?math=\frac{\partial}{\partial\phi_k}f(\phi_k) = \sum_{i} c_i * f(a_i * \phi_k+s_i)">.
```

Comment:
```suggestion
  <img src="https://render.githubusercontent.com/render/math?math=\frac{\partial}{\partial\phi_k}f(\phi_k) = \sum_{i} c_i * f(a_i * \phi_k + s_i)">.
```
![image](https://user-images.githubusercontent.com/49409390/100159041-10a47300-2e7b-11eb-80f9-a5dab643fc18.png)
Looks cool! Noticed that the plus didn't show (although not sure if the suggestion will fix it)

Reply:
Oh good catch! [Have to go with `%2B` apparently](https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b#gistcomment-3009239).